asv_bench/benchmarks/stat_ops.py

```diff
@@ -99,6 +99,7 @@ class Correlation:
     param_names = ["method"]
 
     def setup(self, method):
+        np.random.seed(0)
```

Comment:
You don't need this random seed. The imported `setup` function sets the random seed for all benchmarks

Reply:
Thanks, will remove